.gitignore

```diff
@@ -35,6 +38,7 @@ massif-*
 
 # Unix-like build artifacts:
 *.o
+*.a
```

Comment:
Is it possible to split this out into another commit in the same PR?

Reply:
I'm not sure how to update (2 commits) after PR.  But I guess I could delete PR and make the changes in 2 commits.  It was annoying to keep removing the files every time I build because it wasn't ignored for cross compiling.  Let me know how to proceed if this is not acceptable?  Thanks

Reply:
Just force push your local branch to your fork after making the changes locally, this will update things here.